pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -1376,6 +1376,86 @@ def test_to_datetime_errors_ignore_utc_true(self):
         expected = DatetimeIndex(["1970-01-01 00:00:01"], tz="UTC")
         tm.assert_index_equal(result, expected)
 
+    # TODO: this is moved from tests.series.test_timeseries, may be redundant
```

Comment:
Is this actually redundant? If so I think should just clean up as part of this instead of comment

Reply:
at the very least it needs to be split up, but @simonjayhawkins requested that splitting be done in separate steps, which I agreed to

Reply:
this is a common issue with moving tests, moving and refactoring/cleaning together is more difficult to review and check blame in the future. so atomic commits are good and just moving is fine.